<!DOCTYPE HTML>
<!--
	Solid State by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>omnideconv | Publications</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header -->
					<header id="header">
						<h1><a href="index.html">omnideconv</a></h1>
						<nav>
							<a href="#menu">Menu</a>
						</nav>
					</header>

				<!-- Menu -->
					<nav id="menu">
						<div class="inner">
							<h2>Menu</h2>
							<ul class="links">
								<li><a href="index.html">Home</a></li>
								<li><a href="publications.html">Publications</a></li>	
								<li><a href="learn.html">Learn</a></li>	
							</ul>
							<a href="#" class="close">Close</a>
						</div>
					</nav>

				<!-- Wrapper -->
					<section id="wrapper">
						<header>
							<div class="inner">
								<h2>omnideconv</h2>
							<p>Enabling the deconvolution of any cell type, tissue, and organism</p>
							</div>
						</header>

						<!-- One -->
						<section id="one" class="wrapper alt style1">
							<div class="inner">

								<h2 class="major">omnideconv</h2>
									<ul>
										<li><a href="https://doi.org/10.1093/bioinformatics/btz363"><b>Next-generation deconvolution of the tumor microenvironment with omnideconv</b></a><br/>Merotto L, Dietrich A, Markus List M, Finotello F.<br/><b>Methods in Cell Biology, Immuno-oncology and Immunotherapy</b> (2025).</li><br/>
										<li><a href="https://doi.org/10.1101/2024.06.10.598226"><b>Benchmarking second-generation methods for cell-type deconvolution of transcriptomic data</b></a><br/> Dietrich A*, Merotto L*, Pelz K, Eder B*, Zackl C, Reinisch K, Edenhofer F, Marini F, Sturm G, List M&#10013;, Finotello F&#10013;.<br/><b>bioRxiv 2024.06.10.598226</b> (preprint).</li><br/>
									</ul>

								<h2 class="major">spacedeconv</h2>
									<ul>
										<li><a href="https://doi.org/10.21203/rs.3.rs-5102166/v1"><b>spacedeconv: deconvolution of tissue architecture from spatial transcriptomic</b></a><br/> Zackl C, Zopoglou M, Stauffer R, Ausserhofer M, Ijsselsteijn ME, Sturm, G, de Miranda NFdCC, Finotello F.<br/><b>Research Square</b> (preprint)</li><br/>
										</ul>
								
								<h2 class="major">immunedeconv</h2>
									<ul>
										<li><a href="https://doi.org/10.1093/bioinformatics/btz363"><b>Comprehensive evaluation of transcriptome-based cell-type quantification methods for immuno-oncology</b></a><br/>Sturm G, Finotello F, Petitprez F, Zhang JD, Baumbach J, Fridman WH, List M, Aneichyk T.<br/><b>Bioinformatics</b>, 35(14):i436-i445 (2019).</li><br/>
										<li><a href="https://doi.org/10.1007/978-1-0716-0327-7_16"><b>Immunedeconv: an R package for unified access to computational methods for estimating immune cell fractions from bulk RNA-sequencing data</b></a><br/>Sturm G, Finotello F, List M.<br/><b>Methods Mol Biol</b>, 2120:223-232 (2020).</li><br/>
										<li><a href="https://doi.org/10.1093/bioadv/vbae032"><b>Making mouse transcriptomics deconvolution accessible with immunedeconv</b></a><br/>Merotto L, Sturm G, Dietrich A, List M, Finotello F.<br/><b>Bioinformatics Advances</b>, 4(1):vbae032 (2024).</li><br/>
									</ul>
								
								<h2 class="major">SimBu</h2>
									<ul>
										<li><a href="https://doi.org/10.1093/bioinformatics/btac499"><b>SimBu: bias-aware simulation of bulk RNA-seq data with variable cell-type composition</b></a><br/>Dietrich A, Sturm G, Merotto L, Marini F, Finotello F, List M.<br/><b>Bioinformatics</b>, 38(Suppl_2):ii141-ii147 (2022).</li><br/>
									</ul>

								<h2 class="major">Deconvolution reviews and chapters</h2>
									<ul>
										
										<li><a href="https://doi.org/10.1038/nrg.2016.67"><b>Computational genomics tools for dissecting tumour-immune cell interactions</b></a><br/>Hackl H, Charoentong P, Finotello F, Trajanoski Z.<br/><b>Nat Rev Genet</b>, 17(8):441-58 (2016).</li><br/>
										<li><a href="https://doi.org/10.1007/s00262-018-2150-z"><b>Quantifying tumor-infiltrating immune cells from transcriptomics data</b></a><br/>Finotello F, Trajanoski Z.<br/><b>Cancer Immunol Immunother</b>, 67(7):1031-1040 (2018).</li><br/>
										<li><a href="https://doi.org/10.1038/s41576-019-0166-7"><b>Next-generation computational tools for interrogating cancer immunity</b></a><br/>Finotello F, Rieder D, Hackl H, Trajanoski Z.<br/><b>Nat Rev Genet</b>, 20(12):724-746 (2019).</li><br/>
										<li><a href="https://doi.org/10.1016/bs.mie.2019.05.056"><b>Deconvoluting tumor-infiltrating immune cells from RNA-seq data using quanTIseq</b></a><br/>Plattner C, Finotello F, Rieder D.<br/><b>Methods Enzymol</b>, 636:261-285 (2020).</li><br/>
										<li><a href="https://doi.org/10.1007/978-1-0716-0327-7_15"><b>In silico cell-type deconvolution methods in cancer immunotherapy</b></a><br/>Sturm G, Finotello F, List M.<br/><b>Methods Mol Biol</b>, 2120:213-222 (2020).</li><br/>
										<li><a href="https://doi.org/10.1038/s43588-021-00038-7"><b>Machine learning for deciphering cell heterogeneity and gene regulation</b></a><br/>Scherer M, Schmidt F, Lazareva O, Walter J, Baumbach J, Schulz MH, List M.<br/><b>Nat Comput Sci</b>, 1, 183â€“191 (2021).</li><br/>
										<li><a href="https://doi.org/10.1016/bs.ircmb.2023.05.002"><b>Next-generation deconvolution of transcriptomic data to investigate the tumor microenvironment</b></a><br/>Merotto L, Zopoglou M, Zackl C, Finotello F.<br/><b>Int Rev Cell Mol Biol</b>, 382:103-143 (2024).</li><br/>

									</ul>

								<h2 class="major">Other publications</h2>
									<ul>
										
										<li><a href="https://doi.org/10.1186/s13073-019-0638-6"><b>Molecular and pharmacological modulators of the tumor immune contexture revealed by deconvolution of RNA-seq data</b></a><br/>Finotello F, Mayer C, Plattner C, Laschober G, Rieder D, Hackl H, Krogsdam A, Loncova Z, Posch W, Wilflingseder D, Sopper S, Ijsselsteijn M, Brouwer TP, Johnson D, Xu Y, Wang Y, Sanders ME, Estrada MV, Ericsson-Gonzalez P, Charoentong P, Balko J, de Miranda NFDCC, Trajanoski Z.<br/><b>Genome Med</b>, 11(1):34 (2019).</li><br/>
										<li><a href="https://doi.org/10.1038/s41598-024-53117-w"><b>Blood transcriptomics analysis offers insights into variant-specific immune response to SARS-CoV-2</b></a><br/>Hoffmann M, Willruth LL, Dietrich A, Lee HK, Knabl L, Trummer N, Baumbach J, Furth PA, Hennighausen L, List M.<br/><b>Sci Rep</b>, 14, 2808 (2024).</li><br/>

									</ul>
							</div>
						</section>

				<!-- Footer -->
					<section id="footer">
						<div class="inner">
							<ul class="copyright">
								<li>Francesca Finotello & Markus List. All rights reserved.</li>
								<li>Design: <a href="http://html5up.net">HTML5 UP</a>.</li>
								<li>Photos: <a href="https://unsplash.com/">Unsplash</a>.
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
