# Welcome to omnideconv

The possibility to computationally infer the cellular composition of heterogeneous samples profiled with (cheap) bulk RNA-seq or DNA methylation is a powerful way to advance our understanding of tissue and organ development as well as diseases. Building on the success of [immunedeconv](https://github.com/icbi-lab/immunedeconv) for in silico deconvolution of immune cells, we envision to build a comprehensive platform called omnideconv, which will streamline existing methods for cell type deconvolution, facilitate important benchmarks, offer cross-species support, bridge RNA-seq and DNAm assays and offer user-friendly access to signature generation and comparative analyses. The change in the project's name from immunedeconv to omnideconv is owed to the possibilites offered by second-generation methods which construct cell-type specific signatures directly from single-cell RNA-seq data, thus extending the possibilities of cell-type deconvolution to additional cell types and tissues.

# Contact

If you have questions or comments, reach out to [Francesca Finotello](mailto:francesca.finotello@i-med.ac.at?subject=omnideconv),  [Markus List](mailto:markus.list@wzw.tum.de?subject=omnideconv) or  [Gregor Sturm](mailto:gregor.stum@i-med.ac.at?subject=omnideconv).

